hardware/ariadne/bootloaders/ariadne/main.c

```diff
@@ -135,6 +137,7 @@ int main(void)
 	}
 
 	/* Exit to user application */
+	wdt_reset(); // Just to give time for the user app to start
```

Comment:
You are right, for my app it makes sense to not disable it, but on a general use it indeed should be off. Should I make a new commit ?

About the distance... well you will probably be surprised. It's located more then 60km from my place. I access it using a vpn and it goes trhourg 2 radio links with a span of 20km each.... Rest assured the upload code is sound.

If u re felling curious you can see the information it posts on www.rampasonline.com.br/condicao